src/config.rs

```diff
@@ -199,7 +199,7 @@ pub struct Config {
     config_path: Option<PathBuf>,
 }
 
-static DEFAULT_ALACRITTY_CONFIG: &'static str = include_str!("../alacritty.yml");
+pub static DEFAULT_ALACRITTY_CONFIG: &'static str = include_str!("../alacritty.yml");
```

Comment:
This shouldn't be `pub`. The refactor suggested below would remove the need for it anyway.